dask_kubernetes/helm.py

```diff
@@ -131,8 +139,12 @@ async def _get_scheduler_address(self):
             host = lb.hostname or lb.ip
             return f"tcp://{host}:{port}"
         elif service.spec.type == "NodePort":
-            nodes = await self.core_api.list_node()
-            host = nodes.items[0].status.addresses[0].address
+            if self.nodeport_host:
+                host = self.nodeport_host
+                port = self.node_port
```

Comment:
You check if host is set here, but what if port is set without host?

Reply:
You are right! Adjusted